### PR TITLE
LPD-42542 Add View Usages to the actions menu.

### PR DESCRIPTION
**Pull request by gergelyszalay**
[LPD-42542](https://liferay.atlassian.net/browse/LPD-42542) Add View Usages to the actions menu.

What is this trying to solve?
First PR of [LPD-36446](https://liferay.atlassian.net/browse/LPD-36446), Adding Feature Flag and adding View Usage manu to the document library Adction menu

[Figma](https://www.figma.com/design/2ZnVThOfeekSK3DwnsVv8T/LPD-36446-View-Usages-in-Documents-%26-Media?node-id=4-6539&node-type=frame&t=QETtVvWOy7QDzgBa-0)

How can you verify that it works?
run integration tests in DefaultDLViewFileVersionDisplayContextTest

Commits
[LPD-42538 add LPD-36446 FF](https://github.com/liferay-content-management/liferay-portal/commit/7097154d95150074ebde10a49f06b3fd77ff94d3)
[LPD-42542 Add View Usages to the actions menu.](https://github.com/liferay-content-management/liferay-portal/commit/96d6b87ab76c37a47c085a11e82052be2cb6eea7)
[LPD-42542 Integration test](https://github.com/liferay-content-management/liferay-portal/commit/444158078ff89780063cbb2c34cf549ae9ac913a)

### CODE REVIEW

File: modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/display/context/test/DefaultDLViewFileVersionDisplayContextTest.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+			PermissionCheckerMethodTestRule.INSTANCE,
+			SynchronousDestinationTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
+		_company = _companyLocalService.getCompany(
+			TestPropsValues.getCompanyId());
+		_fileEntry = _addDLFileEntry(
+			RandomTestUtil.randomString(), RandomTestUtil.randomString());
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/display/context/test/DefaultDLViewFileVersionDisplayContextTest.java:88]: Can you sort these alphabetically?

File: modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/display/context/test/DefaultDLViewFileVersionDisplayContextTest.java [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+			content.getBytes(), null, null, null,
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId()));
+	}
+
+	private LayoutClassedModelUsage _addLayoutClassedModelUsage(
+			FileEntry fileEntry)
+		throws Exception {
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext();
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/display/context/test/DefaultDLViewFileVersionDisplayContextTest.java:128]: This is used once; can you inline it?

File: modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/display/context/test/DefaultDLViewFileVersionDisplayContextTest.java [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+		HttpServletRequest mockHttpServletRequest =
+			new MockHttpServletRequest();
+
+		mockHttpServletRequest.setAttribute(
+			ItemSelector.class.getName(), _itemSelector);
+
+		String portletName = RandomTestUtil.randomString();
+
+		String attributeName = StringBundler.concat(
+			portletName, StringPool.DASH, WebKeys.CURRENT_PORTLET_URL);
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/display/context/test/DefaultDLViewFileVersionDisplayContextTest.java:159]: This seems to be used only once; can you inline it?

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:reevaluate:1277401_13192

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:stop

**Comment by jkappler** [comment]
ci:forward

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:forward

**Comment by gergelyszalay** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:test:stable

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:test:stable

**Comment by gergelyszalay** [comment]
ci:report:85102646

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:report:85740432

**Comment by gergelyszalay** [comment]
ci:report:85740432

**Comment by gergelyszalay** [comment]
ci:test:stable

**Comment by gergelyszalay** [comment]
ci:test:stable

**Comment by adolfopa** [comment]
ci:forward

**Comment by gergelyszalay** [comment]
ci:reevaluate:1377401_19100

**Comment by adolfopa** [comment]
ci:test:stable

**Comment by adolfopa** [comment]
ci:forward:force

**Comment by gergelyszalay** [comment]
Included this PR in https://github.com/liferay-content-management/liferay-portal/pull/6162, i will close this

**Pull request closed without merging**